README.md

```diff
@@ -36,13 +36,16 @@ packages:
 3. Add an on-run-end hook to your `dbt_project.yml`: `on-run-end: "{{ dbt_artifacts.upload_results(results) }}"`
 (We recommend adding a conditional here so that the upload only occurs in your production environment, such as `on-run-end: "{% if target.name == 'prod' %}{{ dbt_artifacts.upload_results(results) }}{% endif %}"`)
 
-4. Create the tables dbt_artifacts uploads to with `dbt run-operation create_dbt_artifacts_tables`
+4. In case you don't want the artifacts to be uploaded for certain dbt commands (for instance 'dbt compile'), add the following conditional:
+`on-run-end: "{% if not var('artifact', False) %}{{ dbt_artifacts.upload_results(results) }}{% endif %}"`
```

Comment:
```suggestion
`on-run-end: "{% if not var('dbt_artifacts_upload_results', False) %}{{ dbt_artifacts.upload_results(results) }}{% endif %}"`
```

Cool if we rename this var?

Reply:
yes, I have never been good at naming variables :)